MOBILE_CLIENT/IOS/_IMPORTANT/IPA_ENCRYPTED/meta.json

```diff
@@ -2,9 +2,6 @@
   "risk_rating": "important",
   "short_description": "Application is encrypted by Apple, limiting analysis capabilities.",
   "references": {
-    "Decryption apps guide": "https://kov4l3nko.github.io/blog/2016-03-01-decrypting-apps-from-appstore/",
-    "Repackaging applications guide": "https://labs.mwrinfosecurity.com/blog/repacking-and-resigning-ios-applications/",
-    "lldb-dumpdecrypted": "https://github.com/BlueCocoa/dumpdecrypted-lldb",
```

Comment:
why deleting those?

Reply:
they are dead

Reply:
@Zakaria-Latif Try to find other references, cause now we have None!